workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/GetAppModal/GetAppModal.tsx

```diff
@@ -0,0 +1,196 @@
+import ClayButton from '@clayui/button';
+import ClayModal, {useModal} from '@clayui/modal';
+
+import closeIcon from '../../assets/icons/close-icon.svg';
+import infoCircleIcon from '../../assets/icons/info-circle-icon.svg';
+
+import './GetAppModal.scss';
+import {
+	getChannelById,
+	getProductSKU,
+	patchOrderByERC,
+	postCartByChannelId,
+	postCheckoutCart,
+} from '../../utils/api';
+
+interface GetAppModalProps {
+	account: {
+		id?: number;
+		name: string;
+		email: string;
+		image: string;
+	};
+	app: {
+		name: string;
+		version: string;
+		price: number;
+		image: string;
+		createdBy: string;
+		externalReferenceCode?: string;
+		id: number;
+	};
+	channelId: number;
+}
+
+export function GetAppModal({account, app, channelId}: GetAppModalProps) {
+	const {observer, onOpenChange} = useModal();
+
+	async function handleGetApp() {
+		const channel = await getChannelById(channelId);
+
+		const skuResponse = await getProductSKU({appProductId: app.id});
+
+		const defaultSku = skuResponse.items.find(({sku}) => sku === 'default');
+
+		const newCart: Partial<Cart> = {
```

Comment:
@danvb112 Please, follow the alphabetical order with the objects' properties below.